src/tests/utils/ApiTestSteps.ts

```diff
@@ -510,3 +513,23 @@ export function validateCriVcIssuedTxMAEvent(txmaEvent: any, yotiMockId: any): a
 			console.warn("Yoti Mock Id provided does not match expected list");
 	}
 } 
+
+function validateCriVcIssuedFailedChecks(txmaEvent: any, yotiMockId: any, vcData: any) {
+	// Contra Indicators
+	const expectedContraIndicatiors = eval("vcData.s" + yotiMockId + ".ci");
```

Comment:
This is using the string `vcData` rather than the `vcData` that's passed in, is that what we're meaning to do?

Also could we please add a return type `void` to the function? I'm seeing a ts error

Reply:
Return type added and use of vcData string is intended. Using the eval method allows us to plug variables (yotiMockId) into a string to query the vcData object

Reply:
Oh cool thanks for explaining, is that the same as doing something like vcData[s]?